homeassistant/components/miele/sensor.py

```diff
@@ -930,6 +1028,45 @@ def _update_last_value(self) -> None:
             self._last_value = current_value
 
 
+class MieleAbsoluteTimeSensor(MieleRestorableSensor):
+    """Representation of absolute time sensors handling precision correctness."""
+
+    _previous_value: StateType | datetime = None
+
+    def _restore_last_value(self, last_value: str) -> None:
+        """Restore the last value from cache."""
+        self._last_value = datetime.fromisoformat(last_value)
+
+    def _update_last_value(self) -> None:
+        """Update the last value of the sensor."""
+        current_value = self.entity_description.value_fn(self.device)
+        current_status = StateStatus(self.device.state_status)
+
+        # return cached value if differs only of 90s as debounching,
+        # since API is reporting at minute precision, in order to
+        # avoid changing value too frequently, or if program ended
```

Comment:
This is a little bit hard to understand and "debouncing" is misspelt, maybe:
```suggestion
        # The API reports with minute precision, to avoid changing
        # the value too often, we keep the cached value if it differs
        # less than 90s from the new value
```